src/app/admin/components/audio-file/set-current-version/set-current-version.component.html

```diff
@@ -0,0 +1,83 @@
+<app-govuk-heading> Are you sure you want to make this the current version of the audio file? </app-govuk-heading>
+
+@let audioFile = newVersion();
+
+@if (audioFile) {
+  <div govukSummaryContainer>
+    <app-govuk-heading class="no-margin border-bottom" tag="h2" size="m">Audio details</app-govuk-heading>
+    <dl govukSummaryList>
+      <div govukSummaryListRow>
+        <dt govukSummaryKey>Audio ID</dt>
+        <dd govukSummaryValue>{{ audioFile.id }}</dd>
+      </div>
+
+      <div govukSummaryListRow>
+        <dt govukSummaryKey>Courthouse</dt>
+        <dd govukSummaryValue>{{ audioFile.courthouse.displayName }}</dd>
+      </div>
+
+      <div govukSummaryListRow>
+        <dt govukSummaryKey>Courtroom</dt>
+        <dd govukSummaryValue>{{ audioFile.courtroom.name }}</dd>
+      </div>
+
+      <div govukSummaryListRow>
+        <dt govukSummaryKey>Start time</dt>
+        <dd govukSummaryValue>{{ audioFile.endAt | luxonDate: "dd MMM yyyy 'at' h:mm:ssa" }}</dd>
+      </div>
+
+      <div govukSummaryListRow>
+        <dt govukSummaryKey>End time</dt>
+        <dd govukSummaryValue>{{ audioFile.endAt | luxonDate: "dd MMM yyyy 'at' h:mm:ssa" }}</dd>
+      </div>
+
+      <div govukSummaryListRow>
+        <dt govukSummaryKey>Channel number</dt>
+        <dd govukSummaryValue>{{ audioFile.channel }}</dd>
+      </div>
+
+      <div govukSummaryListRow>
+        <dt govukSummaryKey>Total channels</dt>
+        <dd govukSummaryValue>{{ audioFile.totalChannels }}</dd>
+      </div>
+
+      <div govukSummaryListRow>
+        <dt govukSummaryKey>Media type</dt>
+        <dd govukSummaryValue>{{ audioFile.mediaType }}</dd>
+      </div>
+
+      <div govukSummaryListRow>
+        <dt govukSummaryKey>File type</dt>
+        <dd govukSummaryValue>{{ audioFile.mediaType }}</dd>
+      </div>
+
+      <div govukSummaryListRow>
+        <dt govukSummaryKey>File size</dt>
+        <dd govukSummaryValue>
+          @if (audioFile.fileSizeBytes < bytesInMb) {
+            {{ audioFile.fileSizeBytes | bytes: 'KB' | number: '1.0-2' }}KB
+          } @else {
+            {{ audioFile.fileSizeBytes | bytes: 'MB' | number: '1.0-2' }}MB
+          }
```

Comment:
Do we have this logic anywhere else? Feel like maybe it should be in a component on it's own?